src/components/Explore/TimeSelector.tsx

```diff
@@ -0,0 +1,179 @@
+import { useOnClickOutside } from 'hooks/useOnClickOutside'
+import { useRef } from 'react'
+import React, { useState } from 'react'
+import { Check, ChevronDown, ChevronUp } from 'react-feather'
+import { useModalOpen, useToggleModal } from 'state/application/hooks'
+import { ApplicationModal } from 'state/application/reducer'
+import styled, { css } from 'styled-components/macro'
+
+const TIME_DISPLAYS: { [key: string]: string } = {
+  hour: '1H',
+  day: '1D',
+  week: '1W',
+  month: '1M',
+  year: '1Y',
+}
+
+const enum TimeOption {
+  Hour = 'hour',
+  Day = 'day',
+  Week = 'week',
+  Month = 'month',
+  Year = 'year',
+}
+const TIMES = [TimeOption.Hour, TimeOption.Day, TimeOption.Week, TimeOption.Month, TimeOption.Year]
+
+export enum FlyoutAlignment {
```

Comment:
doesn't need to be exported

Reply:
generally better to only export once something is used, that way you know in future refactors if you remove all usage in a file, the non-exported type can be removed.